src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -41,15 +41,15 @@ internal static class LogRecordExtensions
             };
             request.ResourceLogs.Add(resourceLogs);
 
-            var instrumentationLibraryLogs = new OtlpLogs.InstrumentationLibraryLogs();
-            resourceLogs.InstrumentationLibraryLogs.Add(instrumentationLibraryLogs);
+            var scopeLogs = new OtlpLogs.ScopeLogs();
+            resourceLogs.ScopeLogs.Add(scopeLogs);
```

Comment:
ILogger scope and Otel scopes 🤣

Reply:
There is absolutely no way this will cause confusion. 🙄